src/main/msc/emfat_file.c

```diff
@@ -327,8 +337,16 @@ void emfat_init_files(void)
     emfat_entry_t *entry;
     memset(entries, 0, sizeof(entries));
 
+#ifdef USE_PERSISTENT_MSC_RTC
+    if (mscRebootRtc > 0) { // Only supporting dates after the unix epoch start
+        const int32_t rtcSeconds = rtcTimeGetSeconds(&mscRebootRtc) + (timeConfig()->tz_offsetMinutes * 60);
```

Comment:
Just tested this under linux, and the offset does not work as it should (local time when testing 20181202, 1155 NZDT):

![image](https://user-images.githubusercontent.com/4742747/49333854-5848e900-f62b-11e8-9983-4f42c098b3f5.png)

Setting `timezone_offset_minutes` to `0` will make the file timestamps to appear correct - could it be that they are meant to be UTC?

Reply:
I just retested on OS X, Windows 7 and Windows 10 and all show the correct local time for the file timestamps with my -300 offset.

But in these cases I set the time in the flight controller with an OS X build of the configurator and then connected the USB device to VM's running windows. Don't have a way to build Windows versions of the Configurator at this point. Maybe the timebase used to set the time varies by the platform the Configurator is running on? For OS X at least it sets the time as UTC.

Reply:
If I set `timezone_offset_minutes = 0` I get UTC in CLI `status`. If I set `timezone_offset_minutes = 780` I get local time in CLI `status` (like on the screenshot above).

Reply:
Well it sounds like the time from the configurator is consistent as UTC. I installed node.js on a Windows 10 vm and build the configurator from current master and tested completely in Windows. The timestamps were all correct with my -300 offset (as is the time in `status`). Same for OS X.  So maybe the Linux FAT32 filesystem implementation is behaving differently.

Reply:
I did some research and FAT32 has no concept of timezones in the file timestamps. In NTFS the specification is they're stored as UTC, but not in FAT32.

Reply:
Probably just something we have to live with on linux.

Reply:
Googling a little I’m getting the impression that the UTC assumption is intentional as apparently many cameras, etc. store the files as UTC. So it seems a bit of hack to deal with this.

I suppose we could always add some kind of “use UTC for MSC” flag to work around the workaround. Or maybe it’s not worth it for the limited Linux user base?